### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by goanpeca** [brisvag/napari-label-interpolator#2]
While working on https://github.com/conda-forge/napari-label-interpolator-feedstock/pull/1, we found that some packages are being imported in source code but are not currently declared as dependencies.

- dask-core
- napari
- numpy

Could this be updated for the next release?

Cheers!

**Comment by brisvag** [comment]
Hey, thanks for opening the issue! I removed numpy by mistake, but `napari` and `dask` shouldn't be dependencies: one is only used in typing, and the other is an optional import if the user has it installed just to ensure that if napari provides a dask array I'm treating it correctly.

**Comment by goanpeca** [comment]
Hey, so just to confirm, if the user does not install napari, they can still use this plugin for something standalone?

**Comment by brisvag** [comment]
Yes, you can install this plugin as a library and just use the function in your code without having napari installed... I don't expect many users will do this, but it will work :P
